apis/apps/v1alpha1/opsrequest_types.go

```diff
@@ -448,11 +448,26 @@ type HorizontalScaling struct {
 	// +optional
 	Instances []InstanceTemplate `json:"instances,omitempty"`
 
-	// Specifies the names of instances to be scaled down.
-	// This provides control over which specific instances are targeted for termination when reducing the replica count.
+	// Scale out or in the specified instances based on the operator.
 	//
```

Comment:
// Specifies the names of instances to be scaled in or out based on the `operator` field.
//
// Add:
// - Adds the specified instances to the Component's `clusterComponentSpec.offlineInstances` array.
// - Instances are expected to be online and will be moved to offline status.
//
// Delete:
// - Removes the specified instances from the Component's `clusterComponentSpec.offlineInstances` array.
// - Instances should be offline and will be moved to online status.
//
// Overwrite:
// - Replaces the existing instances in the Component's `clusterComponentSpec.offlineInstances` array with the specified instances.
//
// This field determines which instances are included in `clusterComponentSpec.offlineInstances` field, targeted for termination during replica count adjustment.